Civi/Core/SqlTriggers.php

```diff
@@ -120,7 +120,7 @@ public function createTriggers($info, $onlyTableName = NULL) {
           );
           $variables = str_replace($template_params,
             $template_values,
-            \CRM_Utils_Array::value('variables', $value)
+            $value['variables'] ?? ''
```

Comment:
I was trying to figure out if `$value['variables']` is supposed to be an array as opposed to string, and then couldn't find anywhere in universe this is used (although it's a bit hard to grep for such usage). When you run it on a regular install it's _always_ null. It's not documented at https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_triggerInfo, and looks like it was added at https://github.com/civicrm/civicrm-svn/commit/307b1a6726166192b6fc673b6daff2436dbbb72f, and the unit test added there no longer exists.